.circleci/config.yml

```diff
@@ -545,26 +545,53 @@ workflows:
   version: 2
   test_all:
     jobs:
-      - py2_serverInstall_serverTest
-      - py2_integrationTests
+      - py2_serverInstall_serverTest:
+          filters:
+            tags:
+              only: /^v.*/
```

Comment:
Maybe `/^v[0-9]+.*/` for specificity?

(Minor nit, it seems like either the `^` or the `.*` should be unnecessary, depending on the matching behavior on CircleCI.)